docs/operators/add.md

```diff
@@ -0,0 +1,281 @@
+## `add` operator
+
+The `add` operator adds a value to an `entry`'s `body`, `attributes`, or `resource`.
+
+`field` is a [field](/docs/types/field.md) that will be set to `value` or the evaluated expression.
+
+`value` is a static string that will be added to each entry at the field defined by `field`. 
```

Comment:
Since these are defined in the table below, let's remove them. The descriptions don't match up at this point anyways.